code/edge/src/main/java/com/adobe/marketing/mobile/EdgeConstants.java

```diff
@@ -14,7 +14,7 @@
 final class EdgeConstants {
 
 	static final String EDGE_DATA_STORAGE = "EdgeDataStorage";
-	static final String EXTENSION_VERSION = "1.4.0";
+	static final String EXTENSION_VERSION = "2.0.0";
 	static final String EXTENSION_NAME = "com.adobe.edge";
 	static final String FRIENDLY_NAME = "Edge";
 	static final String LOG_TAG = FRIENDLY_NAME;
```

Comment:
can we now remove EventSource and EventType or will you do that in the next PR?

Reply:
Yes, in another PR. It's actually part 3 of the 2.0 adoption.